### ISSUE TITLE
【美食作家王刚】泡椒火锅鸡 - https://youtu.be/2EiNW3pOmeo

### ISSUE THREAD

**Issue by SilvesterDong** [immoonancient/YTSubtitles#1285]
**视频链接**：https://youtu.be/2EiNW3pOmeo

视频时长7:34，预计发布时间为北京时间7月17日晚
这是刚哥最新的视频私享链接，方便大家提前添加字幕，文字稿在下面
文字稿可能会和视频有微小出入，请大家以视频为主。由于刚哥的合同问题，请大家务必不要泄露或分享刚哥未在YouTube发布的内容，包括文字，视频链接等，望大家理解！

# 请翻译以下全部内容

```
标题（翻译主要意思即可，不要超过 100 个字符）
厨师长分享：“泡椒火锅鸡料包”的使用方法，简单快捷又美味

简介
大家好，有朋友问买来的新产品“火锅鸡”调料具体怎么操作，今天就来用四伯养的鸡来拍一个简单做法，“泡椒火锅鸡”顾名思义其中泡椒和火锅为主要味型，泡椒选用“老坛泡红珠子辣椒”，酸辣适中回味无穷，火锅料选用牛油熬制成的微辣不麻的香浓型底料。如果大家买不到我的产品也可以选用“老坛泡椒”加“微辣香浓型”火锅底料来制作此菜，成菜回味十足酸辣鲜香，喜欢的朋友不要错过！

字幕
0:00:00.169,0:00:01.059
哈喽大家好

0:00:01.560,0:00:04.582
最近很多粉丝说这个火锅鸡的调料

0:00:04.582,0:00:06.028
就是不会做

0:00:06.331,0:00:09.280
然后正好我今天就开一份来做

0:00:09.480,0:00:11.616
我家这个鸡这个鸡是我伯爷养的

0:00:12.177,0:00:13.234
这个鸡没有多大

0:00:13.493,0:00:17.534
去年冬至就买起喂了几个月才这么大点

0:00:17.738,0:00:22.113
它光吃不长今天我就杀来吃了

0:00:22.594,0:00:24.309
这一包是底料

0:00:24.460,0:00:25.660
里面有牛油

0:00:25.700,0:00:27.740
有一些辣椒一起炒的

0:00:27.740,0:00:31.102
然后这一包的话是泡椒

0:00:31.714,0:00:35.170
里面有姜有泡椒这个泡椒不是很辣

0:00:36.009,0:00:37.417
先把这个倒出来

0:00:38.394,0:00:41.914
喂了几大个月才两三斤

0:00:42.480,0:00:43.790
不肯长

0:00:44.541,0:00:45.571
那它吃些什么

0:00:45.780,0:00:49.690
还喂得好诶 拿糠饭喂它

0:00:50.435,0:00:53.030
太肥了也要被吃

0:00:53.030,0:00:54.484
太瘦了也不得行

0:00:55.940,0:00:58.324
管它瘦不瘦哦 我啃骨头嘛

0:01:00.556,0:01:03.340
好 我们这个的话

0:01:03.340,0:01:05.597
如果吃的味道重一点把它切开

0:01:05.597,0:01:08.870
吃的味道不重的话就直接放进去可以了

0:01:08.870,0:01:10.546
然后这里面的这个水不要

0:01:12.659,0:01:14.731
这个切开都有盐味噶

0:01:15.057,0:01:16.930
嗯 切开了就更香

0:01:19.073,0:01:21.774
这个刀拿给漆二娃整得好钝哦

0:01:24.170,0:01:26.116
她这一直磨都没磨一下

0:01:27.784,0:01:31.232
她随便什么都用现成

0:01:32.593,0:01:33.886
好 把它宰了

0:01:39.160,0:01:42.631
喊漆二娃来嘛起码这个鸡要宰半个小时

0:01:43.299,0:01:43.873
为啥子啊

0:01:44.170,0:01:45.356
她手脚慢

0:01:46.839,0:01:49.249
是她嘛半边都没宰完

0:01:50.140,0:01:51.180
不至于吧

0:01:51.180,0:01:52.823
害怕不至于

0:01:52.964,0:01:55.500
她手脚慢又害怕搞到手

0:01:56.120,0:01:57.870
好 拿起去喂狗儿

0:02:00.395,0:02:02.825
给个头给你吃听话点啊

0:02:04.534,0:02:06.220
你先吃完一个再吃嘛

0:02:07.301,0:02:08.415
那么贪心

0:02:09.379,0:02:10.046
慢慢吃

0:02:16.756,0:02:18.174
我们用这个笋子来烧

0:02:18.331,0:02:20.111
这个笋子要从中间切一下

0:02:20.220,0:02:22.310
让它把里面的水流出来

0:02:31.860,0:02:32.885
对半切开就是

0:02:36.540,0:02:37.215
好

0:02:37.840,0:02:39.314
可以了 可以准备做了

0:02:39.314,0:02:40.852
这个焯不焯下水啊

0:02:40.908,0:02:41.334
要

0:04:36.200,0:04:39.607
好 现在我们就转小火烧半个小时

0:04:39.766,0:04:42.161
然后待会儿提前5分钟把这个放下去

0:04:42.415,0:04:43.728
烧熟就可以吃了

0:04:44.048,0:04:46.159
不用调味只放水就可以了

0:04:51.599,0:04:54.699
好啦 现在我们把这个放进去烧一会儿

0:04:55.074,0:04:55.954
烧5分钟

0:05:12.313,0:05:14.110
好啦现在准备出锅了

0:05:21.267,0:05:22.941
尝一下我的火锅鸡

0:05:22.941,0:05:24.022
诶呀 火锅鸡啊

0:05:24.043,0:05:25.017
泡椒味道的

0:05:25.039,0:05:26.098
这就是泡椒火锅鸡啊

0:05:26.158,0:05:28.558
我没有加那个干辣椒

0:05:29.316,0:05:30.194
烧的笋子噶

0:05:30.420,0:05:30.907
嗯

0:05:31.656,0:05:33.162
我还以为你先切得是苦瓜

0:05:33.162,0:05:34.856
我整点来尝一下看耙没耙哦

0:05:35.102,0:05:35.865
肉都耙了

0:05:35.928,0:05:36.903
好脆哦

0:05:38.555,0:05:39.911
嗯 安逸

0:05:40.356,0:05:41.394
味道怎么样

0:05:41.416,0:05:42.454
可以可以

0:05:44.742,0:05:47.560
你看那个狗儿都开始守嘴了

0:05:47.560,0:05:49.348
你不要慌 我们吃了来多

0:05:49.403,0:05:50.589
吃了的骨头拿给它嘛

0:05:51.876,0:05:53.219
我吃那种肉点的

0:05:53.259,0:05:53.916
拿去这坨

0:05:54.516,0:05:55.867
有了有了有了

0:06:00.276,0:06:03.066
嗯还烧耙了的

0:06:03.813,0:06:04.645
安逸

0:06:05.436,0:06:08.186
我们这个火锅鸡就是说

0:06:08.236,0:06:10.186
你自己想弄什么鸡都可以

0:06:10.186,0:06:11.636
也可以放鸭子下去

0:06:12.116,0:06:13.466
还可以放鹅下去

0:06:14.076,0:06:15.036
烧都没有问题

0:06:15.036,0:06:16.556
也可以放兔子啊之类的

0:06:18.515,0:06:20.056
我下次给你多带几包下来

0:06:20.056,0:06:21.576
你想吃的时候随时开起

0:06:21.796,0:06:22.414
给它煮就是

0:06:22.414,0:06:23.916
但是你要记住不能放盐哦

0:06:24.056,0:06:26.567
不放盐啊 你那个佐料里面就有盐的

0:06:26.567,0:06:29.408
佐料里面盐那些都是调归一了的

0:06:29.408,0:06:30.700
辣椒那些都是调好了的

0:06:31.013,0:06:32.178
什么都不放只加水

0:06:32.200,0:06:34.445
现在整饮食吃跟漆二娃做的

0:06:34.615,0:06:37.284
懒人排骨一样只有那么懒了

0:06:37.456,0:06:38.583
整现成的

0:06:38.960,0:06:40.606
佐料啥样都不要

0:06:40.976,0:06:43.286
放到里面样样佐料都有了

0:06:45.252,0:06:46.929
伯娘吃鸡肉撒 你咬得动

0:06:46.945,0:06:47.644
嗯 我吃了的

0:06:47.793,0:06:48.979
现在的人好安逸嘛

0:06:49.405,0:06:51.839
佐料什么的全部配归一又方便

0:06:51.839,0:06:55.172
袋子一割开挤得锅里面

0:06:55.896,0:06:57.551
一烧起就可以干了

0:06:59.893,0:07:01.876
我们农村还要现去准备

0:07:02.455,0:07:03.607
整这样整那样

0:07:03.607,0:07:05.171
关键是很多料你找不到

0:07:05.335,0:07:05.738
嗯

0:07:07.178,0:07:08.801
好啦 本期视频就到这里

0:07:09.255,0:07:13.454
买过这个的朋友们可以这样子试一下

0:07:13.454,0:07:16.650
就说你们之前不是有很多疑问吗

0:07:16.650,0:07:18.156
这个到底要不要加什么要不要加什么

0:07:18.196,0:07:21.289
其实啥都不用加直接加水煮就可以了

0:07:21.501,0:07:23.484
但是前提是要把鸡肉爆香

0:07:23.656,0:07:25.532
然后把泡椒的颜色爆出来

0:07:25.865,0:07:27.056
然后再把那个

0:07:27.263,0:07:29.510
火锅那个料放进去把它炒化

0:07:29.510,0:07:31.498
再加水烧就行了

0:07:31.712,0:07:32.536
什么都不用加

0:07:32.536,0:07:33.616
记住了啊


```

**Comment by andoris** [comment]
认领

**Comment by andoris** [comment]
0:00:00.169,0:00:01.059
哈喽大家好
Hello everyone

0:00:01.560,0:00:04.582
最近很多粉丝说这个火锅鸡的调料
Many fans said that making the seasoning of hotpot chicken

0:00:04.582,0:00:06.028
就是不会做
Is pretty hard

0:00:06.331,0:00:09.280
然后正好我今天就开一份来做
I will cook it today

0:00:09.480,0:00:11.616
我家这个鸡这个鸡是我伯爷养的
This chicken is raised by my uncle

0:00:12.177,0:00:13.234
这个鸡没有多大
It’s not so big though

0:00:13.493,0:00:17.534
去年冬至就买起喂了几个月才这么大点
It’s figure is still small although we bought and raised it since last winter solstice

0:00:17.738,0:00:22.113
它光吃不长今天我就杀来吃了
It eats a lot but never grows so I cook it today

0:00:22.594,0:00:24.309
这一包是底料
This bag is seasoning base

0:00:24.460,0:00:25.660
里面有牛油
There’s beef tallow inside

0:00:25.700,0:00:27.740
有一些辣椒一起炒的
Cooked with chilis 

0:00:27.740,0:00:31.102
然后这一包的话是泡椒
This is a bag of pickled chili

0:00:31.714,0:00:35.170
里面有姜有泡椒这个泡椒不是很辣
There is ginger and pickled chili inside, it’s not very spicy

0:00:36.009,0:00:37.417
先把这个倒出来
Pour out this first

0:00:38.394,0:00:41.914
喂了几大个月才两三斤
It’s only two or three jin (1 or 1.5kg) even it’s several months old

0:00:42.480,0:00:43.790
不肯长
Hard to grow bigger

0:00:44.541,0:00:45.571
那它吃些什么
What does it usually eat

0:00:45.780,0:00:49.690
还喂得好诶 拿糠饭喂它
We feed it with leftovers

0:00:50.435,0:00:53.030
太肥了也要被吃
it will be cooked too if it grew so fat

0:00:53.030,0:00:54.484
太瘦了也不得行
Same situation if it’s too thin

0:00:55.940,0:00:58.324
管它瘦不瘦哦 我啃骨头嘛
Whatever it’s thin or not, I like gnawing the bone

0:01:00.556,0:01:03.340
好 我们这个的话
Alright let’s do it

0:01:03.340,0:01:05.597
如果吃的味道重一点把它切开
Cut it open if you like heavy taste

0:01:05.597,0:01:08.870
吃的味道不重的话就直接放进去可以了
If don’t, just put it in

0:01:08.870,0:01:10.546
然后这里面的这个水不要
We don’t need water inside

0:01:12.659,0:01:14.731
这个切开都有盐味噶
It contains salty flavor if chopped

0:01:15.057,0:01:16.930
嗯 切开了就更香
Yes, it’s more aromatic when chopped

0:01:19.073,0:01:21.774
这个刀拿给漆二娃整得好钝哦
Qi Erwa made this knife so blunt

0:01:24.170,0:01:26.116
她这一直磨都没磨一下
She didn’t even sharpen it for a bit

0:01:27.784,0:01:31.232
她随便什么都用现成
She likes using the ready-made stuff

0:01:32.593,0:01:33.886
好 把它宰了
Ok, let’s butcher it

0:01:39.160,0:01:42.631
喊漆二娃来嘛起码这个鸡要宰半个小时
It takes at least half an hour if Qi Erwa butchered it

0:01:43.299,0:01:43.873
为啥子啊
Why is that

0:01:44.170,0:01:45.356
她手脚慢
Because she is slow

0:01:46.839,0:01:49.249
是她嘛半边都没宰完
She won’t finish half in this time

0:01:50.140,0:01:51.180
不至于吧
Couldn’t be like this

0:01:51.180,0:01:52.823
害怕不至于
Could be maybe

0:01:52.964,0:01:55.500
她手脚慢又害怕搞到手
She acts slow and is afraid of being hurt

0:01:56.120,0:01:57.870
好 拿起去喂狗儿
Alright, just feed it to dogs

0:02:00.395,0:02:02.825
给个头给你吃听话点啊
Be my guest, be obedient

0:02:04.534,0:02:06.220
你先吃完一个再吃嘛
Eat another one after you finish this 

0:02:07.301,0:02:08.415
那么贪心
Oh don’t be so greedy

0:02:09.379,0:02:10.046
慢慢吃
Eat slowly

0:02:16.756,0:02:18.174
我们用这个笋子来烧
We cook it with this bamboo shoot

0:02:18.331,0:02:20.111
这个笋子要从中间切一下
We cut it in the middle

0:02:20.220,0:02:22.310
让它把里面的水流出来
Let the water inside pour out

0:02:31.860,0:02:32.885
对半切开就是
Just cut it evenly

0:02:36.540,0:02:37.215
好
OK

0:02:37.840,0:02:39.314
可以了 可以准备做了
Alright, let’s do it

0:02:39.314,0:02:40.852
这个焯不焯下水啊
Do we need to blanch it

0:02:40.908,0:02:41.334
要
Sure 

0:04:36.200,0:04:39.607
好 现在我们就转小火烧半个小时
Ok we turn it to low heat for half an hour

0:04:39.766,0:04:42.161
然后待会儿提前5分钟把这个放下去
Then put it in 5 mins before finish

0:04:42.415,0:04:43.728
烧熟就可以吃了
We can it when well-braised 

0:04:44.048,0:04:46.159
不用调味只放水就可以了
No need for extra seasoning, just water is enough

0:04:51.599,0:04:54.699
好啦 现在我们把这个放进去烧一会儿
Ok now we put it in and braise for a while

0:04:55.074,0:04:55.954
烧5分钟
About 5 minutes

0:05:12.313,0:05:14.110
好啦现在准备出锅了
Good, now it’s ready 

0:05:21.267,0:05:22.941
尝一下我的火锅鸡
Try this hotpot chicken I made

0:05:22.941,0:05:24.022
诶呀 火锅鸡啊
Aha hotpot chicken

0:05:24.043,0:05:25.017
泡椒味道的
With pickled chili flavor

0:05:25.039,0:05:26.098
这就是泡椒火锅鸡啊
This is actually pickled chili hotpot chicken

0:05:26.158,0:05:28.558
我没有加那个干辣椒
I didn’t add the dried chili pepper into it

0:05:29.316,0:05:30.194
烧的笋子噶
Braised bamboo shoot

0:05:30.420,0:05:30.907
嗯
hmm

0:05:31.656,0:05:33.162
我还以为你先切得是苦瓜
I thought you prepared bitter gourd first

0:05:33.162,0:05:34.856
我整点来尝一下看耙没耙哦
Let me take some and see whether it’s tender or not

0:05:35.102,0:05:35.865
肉都耙了
The meat is so tender

0:05:35.928,0:05:36.903
好脆哦
And very crunchy

0:05:38.555,0:05:39.911
嗯 安逸
Hmm very nice

0:05:40.356,0:05:41.394
味道怎么样		
How’s the taste?

0:05:41.416,0:05:42.454
可以可以
Good good

0:05:44.742,0:05:47.560
你看那个狗儿都开始守嘴了
That dog is licking it’s mouth now 

0:05:47.560,0:05:49.348
你不要慌 我们吃了来多
Don’t worry, we’ve eat a lot

0:05:49.403,0:05:50.589
吃了的骨头拿给它嘛
We will give you the bones 

0:05:51.876,0:05:53.219
我吃那种肉点的
I’ll eat the one with more meat

0:05:53.259,0:05:53.916
拿去这坨
Take this one

0:05:54.516,0:05:55.867
有了有了有了
Yeah got it got it

0:06:00.276,0:06:03.066
嗯还烧耙了的
It’s a tender one

0:06:03.813,0:06:04.645
安逸
Nice

0:06:05.436,0:06:08.186
我们这个火锅鸡就是说
Speaking of this hotpot chicken 

0:06:08.236,0:06:10.186
你自己想弄什么鸡都可以
Any breed of chicken is ok

0:06:10.186,0:06:11.636
也可以放鸭子下去
Ducks are fine 

0:06:12.116,0:06:13.466
还可以放鹅下去
Gooses are good too

0:06:14.076,0:06:15.036
烧都没有问题
We can braise them well

0:06:15.036,0:06:16.556
也可以放兔子啊之类的
Same with rabbit meat

0:06:18.515,0:06:20.056
我下次给你多带几包下来
I’ll bring you more bags of seasoning next time

0:06:20.056,0:06:21.576
你想吃的时候随时开起
You can cook it whenever you want

0:06:21.796,0:06:22.414
给它煮就是
When you are cooking

0:06:22.414,0:06:23.916
但是你要记住不能放盐哦
Remember don’t add salt

0:06:24.056,0:06:26.567
不放盐啊 你那个佐料里面就有盐的
Since seasoning already contains salt

0:06:26.567,0:06:29.408
佐料里面盐那些都是调归一了的
Everything is balanced in this seasoning

0:06:29.408,0:06:30.700
辣椒那些都是调好了的
Including spiciness

0:06:31.013,0:06:32.178
什么都不放只加水
We only need to add water

0:06:32.200,0:06:34.445
现在整饮食吃跟漆二娃做的
Now i cook like Qi Erwa

0:06:34.615,0:06:37.284
懒人排骨一样只有那么懒了
cook the ribs like very lazy process

0:06:37.456,0:06:38.583
整现成的
All ready-made

0:06:38.960,0:06:40.606
佐料啥样都不要
No seasoning at all

0:06:40.976,0:06:43.286
放到里面样样佐料都有了
Just put it in and get everything 

0:06:45.252,0:06:46.929
伯娘吃鸡肉撒 你咬得动
Aunt want to eat hotpot chicken? Is it too tough to bite?

0:06:46.945,0:06:47.644
嗯 我吃了的
Yes I can bite it

0:06:47.793,0:06:48.979
现在的人好安逸嘛
So nice 

0:06:49.405,0:06:51.839
佐料什么的全部配归一又方便
All seasoning in one bag, very convenient

0:06:51.839,0:06:55.172
袋子一割开挤得锅里面
Open the bag and pour it in the wok

0:06:55.896,0:06:57.551
一烧起就可以干了
Heat up and we can eat

0:06:59.893,0:07:01.876
我们农村还要现去准备
We have to prepare everything in our village

0:07:02.455,0:07:03.607
整这样整那样
Like everything

0:07:03.607,0:07:05.171
关键是很多料你找不到
You can’t even find some of the seasoning

0:07:05.335,0:07:05.738
嗯
ok

0:07:07.178,0:07:08.801
好啦 本期视频就到这里
Alright, this episode is about it

0:07:09.255,0:07:13.454
买过这个的朋友们可以这样子试一下
Followers who have bought it could try this way 

0:07:13.454,0:07:16.650
就说你们之前不是有很多疑问吗
You guys had a lot of questions before

0:07:16.650,0:07:18.156
这个到底要不要加什么要不要加什么
Like should we add this seasoning or not

0:07:18.196,0:07:21.289
其实啥都不用加直接加水煮就可以了
Basically we don’t need to add anything except water

0:07:21.501,0:07:23.484
但是前提是要把鸡肉爆香
But we should bring out the aroma of the chicken at first

0:07:23.656,0:07:25.532
然后把泡椒的颜色爆出来
And bring out the color and flavor of the pickled chili

0:07:25.865,0:07:27.056
然后再把那个
And then cook that

0:07:27.263,0:07:29.510
火锅那个料放进去把它炒化
hotpot seasoning till it melt

0:07:29.510,0:07:31.498
再加水烧就行了
Then we just need to add water and heat up

0:07:31.712,0:07:32.536
什么都不用加
With nothing extra

0:07:32.536,0:07:33.616
记住了啊
Remember guys
